geometry_msgs/Vector3Stamped[] FootLocations
std_msgs/Float64 force
